nupic/data/record_stream.py

```diff
@@ -67,19 +67,19 @@ def getNextRecordDict(self):
       '_category': The value from the category field (if any)
       '_reset': True if the reset field was True (if any)
       '_sequenceId': the value from the sequenceId field (if any)
-    
+
     """
-    
+
     values = self.getNextRecord()
     if values is None:
       return None
-    
+
     if not values:
       return dict()
-    
+
     # Create the return dict
-    result = dict(zip(self.getFieldNames(), values))
-    
+    result = dict(list(zip(self.getFieldNames(), values)))
```

Comment:
I think that this would have still worked correctly as `result = dict(zip(self.getFieldNames(), values))`, without the unnecessary overhead of instantiating and populating the `list` object.